File: app/src/main/java/com/BlizzardArmory/warcraft/account/Character_.java

package com.BlizzardArmory.warcraft.account;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Character_ {

    @SerializedName("href")
    @Expose
    private String href;

    public String getHref() {
        return href;
    }

    public void setHref(String href) {
        this.href = href;
    }

}

File: app/src/main/java/com/BlizzardArmory/warcraft/media/Character.kt
package com.BlizzardArmory.warcraft.media

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Character {
    @SerializedName("key")
    @Expose
    var key: Key? = null

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("id")
    @Expose
    var id: Long = 0

    @SerializedName("realm")
    @Expose
    var realm: Realm? = null

}